test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/OpenApiAnyFactoryTests.cs

```diff
@@ -26,6 +26,18 @@ public void CreateFromJson_SimpleType(string json, Type expectedType, object exp
             Assert.Equal(expectedValue, actualValue);
         }
 
+        [Theory]
+        [InlineData("null", typeof(OpenApiNull), null)]
```

Comment:
As there's currently only one case, and I can't really think of a need for others, I would define this test as a `Fact` (no need for `InlineData` plus parameters) rather than a `Theory`

Reply:
Makes sense, will do!